docs/source/index.rst

```diff
@@ -1,16 +1,22 @@
-SkyPilot
-===================================
+SkyPilot Documentation
+======================
 
-SkyPilot is a framework for easily running machine learning projects on any cloud through a unified interface.
+.. figure:: ./images/skypilot-wide-light-1k.png
+  :width: 60%
+  :align: center
+  :alt: SkyPilot
+
+SkyPilot is a framework for easily running machine learning workloads on any cloud through a unified interface. No knowledge of cloud offerings is required or expected – you simply define the workload and its resource requirements, and SkyPilot will automatically execute it on AWS, Google Cloud Platform or Microsoft Azure.
```

Comment:
Wdyt about keeping the first line only, or insert a paragraph break after the first line?

I think people have the tendency to skip paragraphs of text, and chunks of small text are easier to digest.

Reply:
Good point - moved it to a new paragraph.